setup.py

```diff
@@ -18,11 +18,11 @@
         "trie>=1.3.2,<2.0.0",
         "lru-dict>=1.1.6",
     ],
-    # The evm-extras sections is for libraries that the evm does not
+    # The evm-extra sections is for libraries that the evm does not
     # explicitly need to function and hence should not depend on.
     # Installing these libraries may make the evm perform better than
     # using the default fallbacks though.
-    'evm-extras': [
+    'evm-extra': [
```

Comment:
I *did think* about it but couldn't come with something I liked. I'm not crazy about `full` or `complete` because the section in itself is just an *additional part* that *makes* it complete. I kinda like  `evm-batteries` but then that's plural again ¯\_(ツ)_/¯.

Reply:
easy to change later if we find a name we like.  status quo seems fine for now.